circom-prover/src/witness.rs

```diff
@@ -18,7 +19,7 @@ pub enum WitnessFn {
 pub fn generate_witness(
     witness_fn: WitnessFn,
     inputs: HashMap<String, Vec<String>>,
-    _dat_path: String,
+    _zkey_path: String,
```

Comment:
I think we should keep this `_dat_path`.
It is quite common that the zkey name is different from dat name

I am also thinking to make an improvement in `witnesscalc-adapter` to let it find .dat automatically
so it doesn't have to take `.dat` as a function input

Reply:
I created a PR here
https://github.com/zkmopro/witnesscalc_adapter/pull/5
but it might break the current mopro
I think we can merge this after this PR is merged

Reply:
fixed in [183fa83](https://github.com/zkmopro/mopro/pull/308/commits/183fa83bd620e770a6a5dbc477f62cb017ee92ed)